__init__.py

```diff
@@ -622,12 +623,26 @@ def show_date_mark1(self, location, day):
     def get_weekday(self, day=None, location=None):
         if not day:
             day = self.get_local_datetime(location)
-        return day.strftime("%A")
+        weekday = nice_date(day)  # forces lingua_franca to initialize
+        if self.lang in date_time_format.lang_config.keys():
+             weekday = list(
+                 date_time_format.lang_config[self.lang]['weekday'].values())[day.weekday()]
+        else:
+            weekday = day.strftime("%A")
+        return weekday
```

Comment:
It seems these are returned all lowercase, so would need to add a `.capitalize()` for the weekday and month.